src/main/java/org/broadinstitute/consent/http/resources/NihAccountResource.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.consent.http.resources;
+
+import org.broadinstitute.consent.http.models.NIHUserAccount;
+import org.broadinstitute.consent.http.service.NihAuthApi;
+
+import javax.annotation.security.RolesAllowed;
+
+import javax.ws.rs.Path;
+import javax.ws.rs.POST;
+import javax.ws.rs.Produces;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.core.Response;
+
+@Path("api/nih-register/")
+
+public class NihAccountResource extends Resource {
+
+    private NihAuthApi nihAuthApi;
+
+    public NihAccountResource(NihAuthApi nihAuthApi) {
+        this.nihAuthApi = nihAuthApi;
+    }
+
+    @POST
+    @Path("{userId}")
+    @Produces("application/json")
+    @RolesAllowed("RESEARCHER")
+
+    public Response registerResearcher(@PathParam("userId") Integer userId, NIHUserAccount nihAccount) {
```

Comment:
I'm still worried that this takes a userid as a path param. I think it would be much safer to extract this using `@Auth User user`